### PR TITLE
Update tiflash-configuration.md

### PR DESCRIPTION
**Pull request by jinshayumi**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/18555
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -208,6 +208,10 @@ delta_index_cache_size = 0
     ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
     # dt_enable_logical_split = false
 
+    ## `max_threads`: The default value is the number of CPU cores. This parameter
+    ## is the maximum number of threads executed. This parameter only takes effect
+    ## when the system variable `tidb_max_tiflash_threads` is set to -1.
+
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:214]:
```suggestion
    ## For example, if the number of CPU cores is 32, you can set the parameter as follows:
    max_threads = 32
    
```

**Review by hfxsd — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -208,6 +208,12 @@ delta_index_cache_size = 0
     ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
     # dt_enable_logical_split = false
 
+    ## `max_threads`: The default value is the number of CPU cores. This parameter
```
Comment (windtalker) [inline tiflash/tiflash-configuration.md:211]: The default value is `0`, and when it is set to 0, TiFlash will use the number of cpu cores as the execute concurrency.

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -208,6 +208,12 @@ delta_index_cache_size = 0
...
+    ## is the maximum number of threads executed. This parameter only takes effect
```
Comment (windtalker) [inline tiflash/tiflash-configuration.md:212]: This parameter is the internal execute concurrency when TiFlash execute a MPP task.

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -208,6 +208,13 @@ delta_index_cache_size = 0
     ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
     # dt_enable_logical_split = false
 
+    ## `max_threads`: The internal execute concurrency when TiFlash execute a MPP task.
+    ## The default value is 0 and when it is set to 0,
+    ## TiFlash will use the number of cpu cores as the execute concurrency.
+    ## This parameter only takes effect
+    ## when the system variable `tidb_max_tiflash_threads` is set to -1.
+    max_threads = 0
```
Comment (hfxsd) [inline tiflash/tiflash-configuration.md:216]:
```suggestion
    ## `max_threads`: The internal execution concurrency when TiFlash executes an MPP task.
    ## The default value is 0. When it is set to 0,
    ## TiFlash uses the number of CPU cores as the execution concurrency.
    ## This parameter only takes effect
    ## when the system variable `tidb_max_tiflash_threads` is set to -1.
    max_threads = 0
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -208,6 +208,13 @@ delta_index_cache_size = 0
     ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
     # dt_enable_logical_split = false
 
+    ## `max_threads`: The internal execution concurrency when TiFlash executes an MPP task.
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:211]:
```suggestion
    ## `max_threads` indicates the internal thread concurrency when TiFlash executes an MPP task.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by windtalker — APPROVED** [review]
LGTM

**Pull request merged**